app/assets/javascripts/templates/note.ejs

```diff
@@ -1,4 +1,4 @@
-<%= window.md.makeHtml(note.escape("note")) %>
+<%- window.md.makeHtml(note.escape("note")) %>
```

Comment:
Yes, otherwise it won't render the HTML generated by MarkDown. It happened because the EJS lib we used before always rendered it without escaping.